From 12e2693a4868c7f925ad528bb1dafd15d80616bb Mon Sep 17 00:00:00 2001
From: Tycho Andersen <tycho@docker.com>
Date: Mon, 13 Mar 2017 22:44:19 +0000
Subject: [PATCH 1/3] auditctl: include headers to make build work with musl

technically select is defined in sys/select.h, and `struct timeval`
requires sys/time.h

Signed-off-by: Tycho Andersen <tycho@docker.com>
---
 src/auditctl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/auditctl.c b/src/auditctl.c
index e112b16..11d2dc7 100644
--- a/src/auditctl.c
+++ b/src/auditctl.c
@@ -32,6 +32,8 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <sys/utsname.h>
+#include <sys/select.h>
+#include <sys/time.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <libgen.h>	/* For basename */
-- 
2.11.1

